data/scenarios/Testing/1034-custom-attributes.yaml

```diff
@@ -22,8 +22,7 @@ attrs:
     fg: "#ff0000"
     bg: ffff00
   - name: cyanOnMagenta
-    fg: "#00ffff
-    bg: "#ff00ff"
+    fg: "#00ffff bg: " #ff00ff"
```

Comment:
It looks like there was a missing double quote.  I don't know if it worked before, but it certainly doesn't now! :smile:

Reply:
Fix in #1712.